Content.Server/Animals/Components/EggLayerComponent.cs

```diff
@@ -1,36 +1,35 @@
 using Content.Shared.Storage;
 using Robust.Shared.Audio;
-using Robust.Shared.Prototypes;
-using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
 
 namespace Content.Server.Animals.Components;
 
 /// <summary>
 ///     This component handles animals which lay eggs (or some other item) on a timer, using up hunger to do so.
 ///     It also grants an action to players who are controlling these entities, allowing them to do it manually.
 /// </summary>
+
 [RegisterComponent]
 public sealed partial class EggLayerComponent : Component
 {
-    [DataField("eggLayAction", customTypeSerializer: typeof(PrototypeIdSerializer<EntityPrototype>))]
+    [DataField]
     public string EggLayAction = "ActionAnimalLayEgg";
```

Comment:
EntProtoId

Reply:
if you remove the serializer it wont catch invalid ids
EntProtoId does this automatically while string needs the type serializer